snowpack/src/commands/build.ts

```diff
@@ -270,7 +270,6 @@ class FileBuilder {
         const isBundling = !!this.config.experiments.optimize?.bundle;
         const isProxyImport =
           importExtName &&
-          (file.baseExt === '.js' || file.baseExt === '.html') &&
```

Comment:
this is safe to remove now that the CSS import resolver now filters out "proxy imports" automatically. This was only added for that reason.